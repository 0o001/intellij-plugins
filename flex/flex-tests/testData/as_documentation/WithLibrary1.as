package {
public class WithLibrary1 {
    public function WithLibrary1() {
        var t  : Library<caret>Main;
    }
}
}