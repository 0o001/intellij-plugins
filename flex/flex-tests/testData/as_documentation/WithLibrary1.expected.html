<div class='definition'><pre>LibraryMain</pre></div><div class='content'><p>This is library main class</div><table class='sections'></table>